src/scss/coral-talk-iframe/main.scss

```diff
@@ -116,41 +110,220 @@ a {
 	color: oButtonsGetColor('', 'color', 'primary') !important;
 	background-color: oButtonsGetColor('', 'background', 'primary') !important;
 	border-color: oButtonsGetColor('', 'border', 'primary') !important;
+
+	&:hover {
+		color: oButtonsGetColor('', 'color', 'primary') !important;
+		background-color: oButtonsGetColor('', 'background', 'primary') !important;
+		border-color: oButtonsGetColor('', 'border', 'primary') !important;
+	}
 }
 
+
+// cancel buttons
+.coral-createReplyComment-cancel,
+.coral-editComment-cancel {
+	@include oButtons;
+
+	color: oButtonsGetColor('', 'color', 'secondary') !important;
+	border-color: oButtonsGetColor('', 'border', 'secondary') !important;
+
+	&:hover {
+		color: oButtonsGetColor('', 'color', 'secondary') !important;
+		border-color: oButtonsGetColor('', 'border', 'secondary') !important;
+	}
+}
+
+
 // spacing and divider between comments
 .coral-comment {
 	padding-bottom: 6px;
 	padding-top: 10px;
 	border-bottom: solid 1px oColorsGetPaletteColor('black-20');
 }
 
+
 // comment author
-.coral-comment-username {
+.coral-comment-username span {
 	@include oTypographyAuthor;
 
 	&:hover {
 		color: oColorsGetPaletteColor('black');
 	}
 }
 
+
+// report button
+.coral-comment-reactButton,
+.coral-comment-reactedButton,
+.coral-comment-replyButton,
+.coral-comment-shareButton,
+.coral-comment-reportButton,
+.coral-featuredComment-reactButton,
+.coral-featuredComment-replies {
+	&:hover {
+		border-color: transparent !important;
+	}
+
+	span,
+	i {
+		color: oColorsGetPaletteColor('teal-50');
+	}
+}
+
+
+.coral-featuredComment-replies div {
+	color: oColorsGetPaletteColor('teal-50');
+}
+
+
+.coral-comment-replyButton,
+.coral-comment-shareButton {
+	// this is required for selected state
+	border-color: transparent !important;
+	background-color: transparent !important;
+
+	&:hover {
+		border-color: transparent !important;
+	}
+}
+
+
+// report button reported state
+.coral-comment-reportedButton {
+	background-color: oColorsGetPaletteColor('teal-50');
+
+	span,
+	i {
+		color: white;
+	}
+}
+
+
+// share comment - copy button
+.coral-sharePopover-copyButotn,
```

Comment:
Minor spelling mistake here, should be `copyButton`

Reply:
It is spelled like that in Coral. Looks like they made a spelling mistake. 

That's why I added the correct version to the next line, as I anticipate that they will correct it at some point.

Reply:
@ker-an I added a comment to that line so that in the future we know why it is there.

Reply:
That is so funny! But good thinking about adding a correct version as well so that when they do fix it, nothing breaks on our end!